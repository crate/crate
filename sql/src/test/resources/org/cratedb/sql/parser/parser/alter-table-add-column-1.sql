ALTER TABLE t1 add column c double, d bigint