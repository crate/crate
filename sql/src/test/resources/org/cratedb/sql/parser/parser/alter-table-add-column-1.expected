org.cratedb.sql.parser.parser.AlterTableNode@7fe3f2e5
name: t1
statementType: ALTER TABLE
objectName: t1
compressTable: false
sequential: false
truncateTable: false
purge: false
defragment: false
truncateEndOfTable: false
updateStatistics: false
updateStatisticsAll: false
indexNameForUpdateStatistics: null
tableElementList: 	
	org.cratedb.sql.parser.parser.TableElementList@4e5fd9fb

	[0]:	
	org.cratedb.sql.parser.parser.ColumnDefinitionNode@76e015ef
	type: DOUBLE
	name: c
	elementType: null
	[1]:	
	org.cratedb.sql.parser.parser.ColumnDefinitionNode@7a5ad0f0
	type: BIGINT
	name: d
	elementType: null